simfit_recoMC_fullAngular.cc

```diff
@@ -574,7 +574,7 @@ void simfit_recoMC_fullAngularBin(int q2Bin, int parity, bool multiSample, uint
   }
 
   
-  c[confIndex]->SaveAs( ("plotSimFit_d/simFitResult_recoMC_fullAngular_" + shortString + "_" + all_years + stat + "_checkProj_newShapeAnInt.pdf").c_str() );
```

Comment:
I would leave a unique directory for the plots (without the newphi subdirectory) since there is already the suffix to distinguish it from older results. If you prefer the subdirectory, please make sure it is created in the [script](https://github.com/CMSKStarMuMu/UML-fit/blob/adaptive_customMinos_bkg/simfit_recoMC_fullAngular.sh#L14).